examples/device-b.py

```diff
@@ -0,0 +1,18 @@
+from time import sleep, time
+
+from robocluster import Device
+
+device = Device('device-b', '224.0.0.64:32464')
+
+@device.task
+async def hello():
+    message = 'Hello World!'
+    while True:
+        await device.publish('hello', message)
+        await device.sleep(1)
+
+@device.every('100ms')
+async def every():
+    await device.publish('every', time())
```

Comment:
It wouldn't be possible for things like `publish()` and `sleep()` to wrap the `await` keyword would it?

Reply:
Unfortunately we can't.

Reply:
I may be able to get this working by running tasks in an [executor](https://docs.python.org/3.5/library/asyncio-eventloop.html?highlight=run_in_executor#executor)